lib/css/exports/_stacks-constants-helpers.less

```
@@ -44,10 +44,12 @@
 //  ============================================================================
 //  $   BORDER RADIUS (br-)
 //  ----------------------------------------------------------------------------
-@br-sm:                 3px;
-@br-md:                 5px;
-@br-lg:                 7px;
-@br-circle:             50%;
+body {
+    --br-sm:                 3px;
+    --br-md:                 5px;
+    --br-lg:                 7px;
+    --br-circle:             50%;
+}
```

```suggestion
@br-sm:                 3px;
@br-md:                 5px;
@br-lg:                 7px;
@br-circle:             50%;
```

We don't need this change in _this_ PR, do we?

---

Whoops!